elasticgraph-graphql/spec/support/sub_aggregation_support.rb

```diff
@@ -16,12 +16,20 @@ module Aggregation
           {"size" => size, "adapter" => grouping_adapter.meta_name}.merge(hash)
         end
 
-        def inner_terms_meta(hash = {})
-          {"merge_into_bucket" => {}}.merge(hash)
+        def inner_terms_meta(grouping_fields, hash = {})
+          {
+            "grouping_fields" => grouping_fields.is_a?(Array) ? grouping_fields : [grouping_fields],
+            "key_path" => ["key"],
+            "merge_into_bucket" => {}
+          }.merge(hash)
         end
 
-        def inner_date_meta(hash = {})
-          {"merge_into_bucket" => {"doc_count_error_upper_bound" => 0}}.merge(hash)
+        def inner_date_meta(grouping_fields, hash = {})
+          {
+            "grouping_fields" => [grouping_fields],
```

Comment:
This can also use `Array(grouping_fields)` if you want to flexibly allow an array or a single value.